frontend/src/components/transactions/types.ts

```diff
@@ -0,0 +1,43 @@
+export interface Transaction {
```

Comment:
These types are available in `nearlib`, but we don't use `nearlib` on the frontend, so I would extract them into a separate package in the future. Ideally, I would prefer having them defined by nearcore public API (JSON RPC is not great at this, so ProtoBuf / Flatbuffers / ... with automatic code-generation is the direction I want to move in future)